### PR TITLE
Feature checkout form

### PR DESCRIPTION
**Pull request by RRicardotj**
Checkout v3: Add checkout page feature

### CODE REVIEW

File: src/MercadoPago/createTokenByDOM.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const createTokenByDOM = async ({ paymentMethodId }) => {
+  var $form = document.querySelector('#mercadopagoCheckoutForm')
+
+  const paymentMethodIdInput = document.createElement('input')
+  paymentMethodIdInput.setAttribute('name', 'paymentMethodId')
+  paymentMethodIdInput.setAttribute('type', 'hidden')
+  paymentMethodIdInput.setAttribute('value', paymentMethodId)
+  $form.appendChild(paymentMethodIdInput)
+
+  const token = await createTokenHandlePromise($form)
```
Comment (abelosorio) [inline src/MercadoPago/createTokenByDOM.js]: Why are we waiting for the promise to be fulfilled? Can't we return it directly?

File: src/MercadoPago/createTokenByDOM.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import createTokenHandlePromise from './createTokenHandlePromise'
+
+const createTokenByDOM = async ({ paymentMethodId }) => {
+  var $form = document.querySelector('#mercadopagoCheckoutForm')
```
Comment (abelosorio) [inline src/MercadoPago/createTokenByDOM.js]: Why are we using var here?

Comment (RRicardotj) [reply src/MercadoPago/createTokenByDOM.js]: oh, my bad.. let me fixed

File: src/MercadoPago/createTokenByDOM.js (hunk 1 above)
Comment (RRicardotj) [reply src/MercadoPago/createTokenByDOM.js]: yes, we can

**Pull request merged**